org/mozilla/jss/tests/all.pl

```diff
@@ -92,7 +93,7 @@ sub checkPort {
 my $run_shell    = "";
 my $pwfile       = "";
 my $nss_bin_dir  = "";
-my $jarFiles     = "";
+my $classpath    = "";
```

Comment:
No need to expose jarFiles; handle this in `setup_vars` and let everything use `classpath`.